pages/agenda/2017.tsx

```diff
@@ -47,6 +47,7 @@ const Agenda2017: React.FC<AgendaPageProps & { sessions: Session[] }> = ({ sessi
               sponsors={From2019.Sponsors}
```

Comment:
Is this alright? `From2019. Sponsors`

Reply:
whoops. That's very not ok. Nice catch!

Reply:
Only is they are the same sponsors, but I doubt